src/admin/settings/settings.html

```diff
@@ -0,0 +1,83 @@
+            <input type="file" id="fileInput" name="file" value="{{'server.button.selectFile' | translate}}"/>
+        </div>
+    </fieldset>
+    <input ng-click="uploadSettings()" type="submit" value="{{'admin.submit' | translate}}" />
+</form>
+<form id="settingsLocationForm" method="post" ng-submit="uploadFileLocation()">
+    <fieldset>
+        <h4>{{'admin.settings.location' | translate}}</h4>
+        <div class="form-group">
+            <label for="settingsLocation">{{'admin.settings.location.new' | translate}}</label>
+            <input type="text" id="settingsLocation" name="settingsLocation" ng-model="location"/>
+        </div>
+    </fieldset>
+    <input type="submit" value="{{'admin.submit' | translate}}"/>
+</form>
+<form>
+    <fieldset>
+        <h4>{{'admin.settings.importexport.header' | translate}}</h4>
+        <div class="form-group">
+            <label for="exportConfig">{{'admin.settings.importexport.export' | translate}}</label>
+            <input class="input-small" type="button" id="exportConfig" name="exportConfig" ng-click="exportConfig()" value="{{'admin.settings.importexport.exportButton' | translate}}" class="btn btn-default"/>
+        </div>
+    </fieldset>
+</form>
+
+<h3>{{'admin.settings.platform' | translate}}</h3>
+<div ng-repeat="pSettings in platformSettings.settingsList">
+    <form id="platformSettingsForm">
+        <h4>{{'admin.settings.section.'+ pSettings.section | translate}}</h4>
+        <fieldset ng-repeat="option in pSettings.settings">
+            <div class="form-group">
+                <label for="{{option.key}}">
+                    {{'admin.settings.' + option.key | translate}}
+                    <span class="icon-documentation" data-trigger="hover" data-placement="left" bs-popover="{{'admin.settings.tooltip.' + option.key | translate}}" />
+                </label>
+                <input ng-if="isTextOrNumber(option)"ng-disabled="platformSettings.readOnly" type="text" id="{{option.key}}" name="{{option.key | translate}}" ng-model="option.value" />
+                <input class="input-small" ng-if="isBool(option)" ng-disabled="platformSettings.readOnly" type="checkbox" id="{{option.key}}" name="{{option.key}}" ng-model="option.value" />
+                <div ng-if="isCombobox(option)" ng-init="addValue(option.key, option.value)">
+                    <select ng-model="option.value" ng-disabled="platformSettings.readOnly" ng-options="value for value in getComboboxValues(option.key)">
+                        <option ng-if="!option.value" value="">{{'mds.form.label.select' | translate}}</option>
+                    </select>
+                    <input ng-disabled="platformSettings.readOnly" type="text" ng-model="userValidator" />
+                    <a class="btn btn-success" ng-disabled="!userValidator" ng-click="addValue(option.key, userValidator)">
+                        <span class="fa fa-fw fa-plus"></span>Add option</a>
```

Comment:
change class to `icon-add` from [common files](https://github.com/motech/motech-ui/blob/master/src/common/base/base.icons.scss)